lib/src/typings.d.ts

```diff
@@ -1,10 +1,23 @@
 declare namespace RPE {
   interface State {
     effectsEntity: Record<string, boolean>;
+    ignoredActions: string[],
+    isIgnoredActionsSetUp: boolean
   }
 
   interface PayloadAction<T, P> {
     type: T;
     payload: P;
   }
+
+  type PatchEffectPayload = {
```

Comment:
I have one more interesting idea.
Can we add the third arg "effectType" which is an enum of `saga`, `toolkit`, `promise`, `thunk` (in future)